dao.py

```diff
@@ -1,34 +1,24 @@
-from flask_pymongo import PyMongo
-from model import Model
+from pymongo import MongoClient
 
 class Dao:
-    def __init__(self, app):
-        app.config['MONGO_DBNAME'] = 'thing_list'
-        
-        # TODO: clusert configuration is good here, but this will connect to a local mongo docker container once you build out
-        # your docker-compose file
-        app.config['MONGO_URI'] = 'mongodb://admin:admin@cluster0-shard-00-00-bwkql.mongodb.net:27017,cluster0-shard-00-01-bwkql.mongodb.net:27017,cluster0-shard-00-02-bwkql.mongodb.net:27017/test?ssl=true&replicaSet=Cluster0-shard-0&authSource=admin'
-        
-        self.mongo = PyMongo(app)
-
+    def __init__(self):
```

Comment:
you're not using mongo as a replicated set so why are you initializing mongo like it ? just connect to a single mongo instance that you will boot up using docker-compose and connect directly to that